db/schema.rb

```diff
@@ -11,18 +11,19 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160618153536) do
+ActiveRecord::Schema.define(version: 20160618190136) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "entries", force: :cascade do |t|
-    t.integer  "sample_id",              null: false
-    t.integer  "taxa_id",                null: false
-    t.integer  "count",      default: 0
-    t.string   "name",                   null: false
-    t.datetime "created_at",             null: false
-    t.datetime "updated_at",             null: false
+    t.integer  "sample_id",                 null: false
+    t.integer  "taxa_id",                   null: false
+    t.integer  "count",         default: 1
+    t.string   "name",                      null: false
+    t.datetime "created_at",                null: false
+    t.datetime "updated_at",                null: false
+    t.hstore   "response_data"
```

Comment:
Arg, this PR wasn't supposed to include the hstore columns. Not the end of the world, though.